components/exercises/chapter-6-oscar/group/MembersGroup.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import { Group } from './Group';
+
+/**
+ * Renders a component that allows to add and remove members from a group.
+ * @returns JSX.Element
+ */
+export function MembersGroup(): JSX.Element {
```

Comment:
If there's only one export in the file, use `export default`

Reply:
Refactored, thank you